src/liballoc/alloc.rs

```diff
@@ -2,9 +2,11 @@
 
 #![stable(feature = "alloc_module", since = "1.28.0")]
 
-use core::intrinsics::{min_align_of_val, size_of_val};
-use core::ptr::{NonNull, Unique};
-use core::usize;
+use core::{
+    intrinsics::{min_align_of_val, size_of_val},
+    ptr::{NonNull, Unique},
+    usize,
+};
```

Comment:
Hm, I'm not sure if we want to use this new path-nested syntax; is there a recommendation from the style team? cc @nrc 

Probably also worth asking @rust-lang/libs what they think.

Reply:
I don't think there are recommendations yet; this is going by @mark-i-m's style in https://github.com/rust-lang/rust/issues/55618#issuecomment-436058768.
I personally find that it makes it clear what things belong to what crate, etc.

Reply:
I would prefer that stylistic changes are done intentionally with broad agreement rather than through a PR.

Reply:
Alright; I'll revert the import style changes then. :)